book/src/docker.md

```diff
@@ -77,8 +72,7 @@ The `stability` is:
 
 The `modernity` is:
 
-* `-modern` for optimized builds
-* empty for a `portable` unoptimized build
+* `-portable` for the default optimized builds
```

Comment:
modernity is gone from the image tag, so we can remove this, and `modernity` from line 54

Reply:
Thank you, removed in [9339a49](https://github.com/sigp/lighthouse/pull/6247/commits/9339a497a4a5616a8fa4a119aede6ae48a339ed1)